spaghetti-core/src/main/java/com/prezi/spaghetti/obfuscation/internal/SymbolCollectVisitor.java

```diff
@@ -54,7 +54,7 @@
 
 	@Override
 	public Collection<String> visitConstNode(ConstNode node) {
-		return extractNames(node.getEntries());
+		return ImmutableList.<String>builder().add(node.getName()).addAll(extractNames(node.getEntries())).build();
```

Comment:
Why not use `ImmutableList.copyOf()`? Or even better, rewrite `extractNames()` to create the immutable collection itself instead?